api/docs/release.dox

```diff
@@ -241,6 +241,8 @@ Further non-compatibility-affecting changes include:
    trace analysis tool framework.
```

Comment:
> Note that we remove the TRACE_MARKER_TYPE_FUNC_ markers related to functions whose ID is not in the list.

This is very important and should not be under "Note": `s/Note that we/We/`.

Reply:
Done.